app/helpers/repo_helper.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module RepoHelper
```

Comment:
For this type of code, the ideal way is to use Services. Helpers are meant for the View layer.

Reply:
Ah, that's what I was looking for. Will change 👍